package.json

```diff
@@ -24,6 +26,13 @@
   },
   "homepage": "https://github.com/alejandro-bp/cypress-training#readme",
   "devDependencies": {
-    "cypress": "^10.3.0"
```

Comment:
make sure you have well configured the version you want to use:
- the cypress.config.js file is in the new format for cypress 10.
- the spec.js format of the tests file names is the used in version 9. but the format in version 10 is .cy.js

The workshop is documented for version 9, but if you want, you can use version 10, just that config files format is a little bit different, but searching in the doc you will find how to handle the issues. If you want to  continue using version 9, then change the cypress.config.js file to use the format of version 9.